modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CSharpNetCoreClientCodegen.java

```diff
@@ -733,6 +770,44 @@ else if (GENERICHOST.equals(getLibrary())){
         addTestInstructions();
     }
 
+    @Override
+    public CodegenOperation fromOperation(String path,
+                                          String httpMethod,
+                                          Operation operation,
+                                          List<Server> servers) {
+        CodegenOperation op = super.fromOperation(path, httpMethod, operation, servers);
+
+        if (!GENERICHOST.equals(getLibrary())){
+            return op;
+        }
+
+        Collections.sort(op.allParams, new Comparator<CodegenParameter>() {
+            @Override
+            public int compare(CodegenParameter one, CodegenParameter another) {
+                if (one.defaultValue == another.defaultValue)
```

Comment:
@devhl-labs should we use `.equals()` to compare the value?

ref: https://sonarcloud.io/project/issues?resolved=false&types=BUG&id=OpenAPITools_openapi-generator&open=AX77TszkotLJCeLQH1lh

Reply:
Probably. Pretty sure we can just remove this if we merge my latest pr.